File: 462-minimum-moves-to-equal-array-elements-ii/462-minimum-moves-to-equal-array-elements-ii.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var minMoves2 = function(nums) {
    let st = 0
    let ed = nums.length-1
    let ans = 0
    nums.sort((a, b) => a-b)
    while(st < ed) {
        ans += nums[ed] - nums[st]
        ed--
        st++
    }
    
    return ans
};

File: 462-minimum-moves-to-equal-array-elements-ii/462-minimum-moves-to-equal-array-elements-ii.cpp
class Solution {
public:
    int minMoves2(vector<int>& nums) {
        int n = nums.size();
        sort(nums.begin(), nums.end());
        
        int med = nums[(n-1)/2];
        int ans = 0;
        for(int itr : nums) {
            ans += abs(itr - med);
        }
        
        return ans;
    }
};